packages/strapi-admin/ee/admin/components/Users/MagicLink/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import LinkNotification from '../../../../../admin/src/components/Users/LinkNotification';
+import basename from '../../../../../admin/src/utils/basename';
+
+const MagicLink = ({ registrationToken }) => {
```

Comment:
What do you think of renaming this `OAuthRegistrationLink`? 😛

Reply:
We added a new case for SSO. This component will display a registration link only if you use the default workflow. If the so toggle is set to ON, it displays a link to the login page. Maybe we can rename it to something like `AuthLink`?